### PR TITLE
Update Production

### PR DESCRIPTION
**Pull request by opadron**
- reverting reversions
 - PyJWT bump

### CODE REVIEW

**Comment by opadron** [comment]
@zackgalbreath feel free to review & merge.

**Comment by opadron** [comment]
Ok, looks like this ended up getting some more pushes.  Let's hold off until #265 merges and then we can prepare to merge this PR.

**Review by zackgalbreath — APPROVED** [review]

**Pull request merged**
